AirCasting/SDSync/MobileSessions/SDSyncAveragingService.swift

```diff
@@ -0,0 +1,77 @@
+// Created by Lunar on 29/09/2022.
+//
+
+import Foundation
+import CoreLocation
+
+protocol SDSyncAveragingService {
+    func averageMeasurementsWithReminder<T: AverageableMeasurement>(measurements: [T], startTime: Date, averagingWindow: AveragingWindow, averagingAction: (T, [T]) -> Void) -> [T]
```

Comment:
we need to provide a documentation here. The method signature itself is pretty clear, but to understand what I'm supposed to provide in an `averagingAction` is not (I mean - if I wanted to implement this function I wouldn't know what I'm supposed to do with the averagingAction parameters at all).